server/text_generation_server/models/causal_lm.py

```diff
@@ -593,6 +670,9 @@ def __init__(
                 model = wrap_in_hpu_graph(model, disable_tensor_cache=True)
         model = self.setup_quantization(model)
 
+        if self.enable_hpu_graph:
+            enable_hpu_graphs()
```

Comment:
This is also confusing.. We are enabling HPU graphs 3 lines above, so this function name is not accurate.